app/test_split_integer.py

```diff
@@ -2,20 +2,30 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(32, 6)) == 32, (
+        "Sum of the parts should be equal to value"
+    )
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert len(set(split_integer(16, 4))) == 1, (
+        "When value divisible by parts split into equal parts"
+    )
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert split_integer(8, 1) == [8], (
+        "Return part equals to value when split into one part"
+    )
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    assert split_integer(16, 4) == sorted(split_integer(16, 4)), (
```

Comment:
the program splits 16 into 4 equal parts - [4,4,4,4]. So the result is always sorted, you need to select different dataset to test this